src/main/java/uk/gov/digital/ho/hocs/workflow/api/WorkflowService.java

```diff
@@ -66,9 +67,39 @@ public WorkflowService(CaseworkClient caseworkClient,
         this.userPermissionsService = userPermissionsService;
     }
 
-    public CreateCaseResponse createCase(String caseDataType, LocalDate dateReceived, List<DocumentSummary> documents, UUID userUUID, UUID fromCaseUUID) {
+    public CreateCaseResponse createCase(String caseDataType, LocalDate dateReceived, List<DocumentSummary> documents, UUID userUUID, UUID fromCaseUUID, Map<String, String> receivedData) {
```

Comment:
Just to confirm, the front end call was changed to pass in the map of `receivedData` by default wasn't it? We should be careful on here.

Reply:
I don't think we need to change anything, only that we make sure the corresponding change goes in to the `hocs-frontend` code.